### DISCUSSION TITLE
5.1.3: multi monitor / multi plank dock setup no longer working.

### DISCUSSION THREAD

**Discussion by overcookedpanda** [elementary/os#676] [Q&A] [answered]
## Prerequisites
- [X] I have searched open and closed issues for duplicates.

## Describe the bug
After updating to 5.1.3, when I boot or login, my primary display had changed and I have 2 docks on a single monitor.

I was able to restore the primary monitor, but am unable to get a second instance of plank running on my 2nd monitor.

I have my `~/.config/plank/dock1` folder linked to `~/.config/plank/dock2` and a settings file inside the `dock2` folder with `Monitor=1`.  I am unsure if I previously had a settings file inside the `dock1` folder, but I do not at this time.

This setup was working without issue prior to the update to 5.1.3 and I had a duplicated dock on each of my monitors.

I have set `sh -c plank & plank -n dock2` to run at boot time to active the dual docks.

### To Reproduce
I followed similar steps in this article for the setup that previously worked:
https://heathpaddock.com/2014/08/25/dual-docks-in-elementary-os/

### Expected behavior
Two plank instances created, with identical dock icons, one on each physical monitor.

## Platform Information
![image](https://user-images.githubusercontent.com/35861207/78711241-47be1800-78dc-11ea-963d-cd6fd789f522.png)

Please check what applies:
- [ ] I'm using the latest version from git that I've manually compiled
- [X] I'm using the latest released stable version

## Additional context
If there is a better solution to getting a duplicated dock running on each of monitors, I am open to it, it would be great if the OS had this functionality built-in.

**Comment by overcookedpanda** [comment, marked as answer]
As an update, I found a workaround, by going into the preferences of one of the docks, I was able to move it to the other monitor, and it seems to persist on logout/login now.

**Comment by akarasulu** [comment]
> As an update, I found a workaround, by going into the preferences of one of the docks, I was able to move it to the other monitor, and it seems to persist on logout/login now.

Can you explain how you did that? Wondering if you used dconf-editor and if so which keys. Thanks!

**Comment by lenemter** [comment]
@akarasulu You can Ctrl+Right click on dock  to open Plank Preferences.
